endoreg_db/views/api/patient_finding_views.py

```
@@ -0,0 +1,317 @@
+from endoreg_db.models import (
+    PatientFinding, 
+    Examination,
+    Finding,
+    FindingLocationClassification,
+    FindingMorphologyClassification
+)
+from .serializers.patient_finding import (
+    PatientFindingDetailSerializer,
+    PatientFindingListSerializer,
+    PatientFindingWriteSerializer
+)
```

Pylint (3.3.7)

[error] Attempted relative import beyond top-level package

(E0402)